.github/workflows/tacos_unlock.yml

```diff
@@ -40,6 +64,10 @@ jobs:
       github.event.label.name == ':taco::unlock'
     outputs:
       summary: ${{ steps.release-terraform-lock.outputs.summary }}
+    needs: [determine-tf-root-modules]
+
+    env:
+      TF_ROOT_MODULES: ${{ needs.determine-tf-root-modules.outputs.slices }}
```

Comment:
```suggestion
      TF_ROOT_MODULE: ${{ needs.determine-tf-root-modules.outputs.slices }}
```

Reply:
Are you sure? 
`TF_ROOT_MODULES` contains the list of the slices, not the path of a single slices. 
The scripts that rely on `TF_ROOT_MODULE` like `default-shell-post-sudo` expect one path.